.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "0.10"
+env:
+  - TEST_DIR=submissions/gterzian/
+script: cd $TEST_DIR && npm install && npm test
```

Comment:
@staltz By the way this could be quite useful, it would allow anyone to run tests for their own directory...